addons/mail/static/src/discuss/core/common/thread_service_patch.js

```diff
@@ -27,4 +27,26 @@ patch(ThreadService.prototype, {
         }
         return super.post(...arguments);
     },
+
+    async fetchMoreAttachments(thread, limit = 30) {
+        if (thread.isLoadingAttachments || thread.areAttachmentsLoaded) {
+            return;
+        }
+        thread.isLoadingAttachments = true;
+        try {
+            const rawAttachments = await this.rpc("/discuss/channel/attachments", {
+                before: Math.min(...thread.attachments.map(({ id }) => id)),
+                channel_id: thread.id,
+                limit,
+            });
+            const attachments = rawAttachments.map((rawAttachment) =>
+                this.store.Attachment.insert(rawAttachment)
+            );
```

Comment:
Maybe a short name for variable in loop could be single line and make prettier is happy?
This is a case where less LOCs can be beneficial thanks to shorter names. Even a plain `data` or `d` is good, it's just a variable in this tiny scope